lib/cassanity/config.rb

```diff
@@ -0,0 +1,22 @@
+
+require 'ostruct'
+require 'yaml'
+require 'singleton'
+require 'erb'
+
+module Cassanity
+  class Config < OpenStruct
+    include Singleton
+
+    CONFIG_FILE = 'config/cassanity.erb.yml'
```

Comment:
Also, where does this file live? Is it in cassanity? Or does it enforce that it should be in this location in the project that includes these tasks?